test/unit/console_log_utils_test.dart

```diff
@@ -129,6 +129,47 @@ void main() {
         });
         expect(printCalls, ['foo', 'bar']);
       });
+
+      test('prints even if the function throws partway through', () {
```

Comment:
I added this extra test to ensure the try-finally was set up properly in the new `printConsoleLogs` implementation